core-bundle/src/EventListener/RequestTokenListener.php

```diff
@@ -51,12 +52,14 @@ public function __invoke(RequestEvent $event): void
         // Only check the request token if
         // - the request is a POST request,
         // - the request is not an Ajax request,
+        // - the request is not a simple CORS Content-Type that does not require a preflight request,
```

Comment:
```suggestion
        // - the request is a CORS Content-Type that requires a preflight request,
```

I find it easier to understand this way. But feel free to ignore ☺️